mdp/nodes/sfa_nodes.py

```diff
@@ -267,25 +273,26 @@ def _stop_training(self, debug=False):
         # not the second central moment matrix (centered about the mean), i.e.
         # the covariance matrix
         if hasattr(self, '_dcov_mtx'):
-            self.dcov_mtx, self.davg, self.dtlen = self._dcov_mtx.fix(center=False)
+            self.dcov_mtx, self.davg, self.dtlen = self._dcov_mtx.fix(
+                center=False)
             del self._dcov_mtx
 
         rng = self._set_range()
 
-        #### solve the generalized eigenvalue problem
+        # solve the generalized eigenvalue problem
         # the eigenvalues are already ordered in ascending order
         try:
             try:
                 # We first try to fulfill the extended signature described
                 # in mdp.utils.symeig_semidefinite
                 self.d, self.sf = self._symeig(
-                        self.dcov_mtx, self.cov_mtx, True, "on", rng,
```

Comment:
From https://www.python.org/dev/peps/pep-0008/#indentation: "When using a hanging indent [...] further indentation should be used to clearly distinguish itself as a continuation line."